wrangler/package.json

```diff
@@ -3,6 +3,8 @@
   "version": "0.0.0",
   "main": "./src/index.js",
   "scripts": {
+    "preinit": "[ \"${CI}\" = true ] && npx envdist -e || npx envdist",
```

Comment:
on GH action read values from environment-variables and on local interactively prompt